help.go

```diff
@@ -39,14 +39,14 @@ func mainUsage(f io.Writer) {
 }
 
 var mainHelpTemplate = `
-The modcop command checks that module dependencies do not stray outside
+The modcop command checks that package dependencies do not stray outside
 well defined boundaries. It expects that the set of allowed dependencies
 will be checked in along with the source code, and modcop will be called
 as part of a CI check, so any addition to the set of allowed dependencies
 will need to be explicitly vetted as part of the code review process.
 
 The set of dependencies checked by modcop does not include dependencies
-of test code outside of the module being checked - it differs from
+of test code outside of the packages being checked - it differs from
```

Comment:
My point from before is that it only makes sense (I think) for the packages argument to `modcop` to be a subset of the main module's packages. As it stands, it appears I can give `modcop` any packages as arguments... and I don't think that's what we're intending.

Reply:
I'm not sure currently. I'll leave it as is for now. Some thought needed in the future.